Cargo.toml

```diff
@@ -1,44 +1,53 @@
 [package]
-name = "tuf"
-version = "0.0.2"
-authors = [ "heartsucker <heartsucker@autistici.org>" ]
+authors = ["heartsucker <heartsucker@autistici.org>"]
+categories = ["cryptography"]
 description = "Library for The Update Framework (TUF)"
-homepage = "https://github.com/heartsucker/rust-tuf"
-repository = "https://github.com/heartsucker/rust-tuf"
 documentation = "https://docs.rs/tuf"
-readme = "README.md"
+homepage = "https://github.com/heartsucker/rust-tuf"
+keywords = ["security"]
 license = "MIT"
-keywords = [ "security" ]
-categories = [ "cryptography" ]
+name = "tuf"
+readme = "README.md"
+repository = "https://github.com/heartsucker/rust-tuf"
+version = "0.0.2"
 
 [badges]
-travis-ci = { repository = "heartsucker/rust-tuf", branch = "master" }
 
-[lib]
-name = "tuf"
-path = "./src/lib.rs"
+[badges.travis-ci]
+branch = "master"
+repository = "heartsucker/rust-tuf"
 
 [[bin]]
+doc = false
 name = "tuf"
 path = "./src/main.rs"
-doc = false
-required-features = [ "cli" ]
-
-[features]
-cli = [ "clap" ]
+required-features = ["cli"]
 
 [dependencies]
-chrono = { version = "0.3", features = [ "serde" ] }
-clap = { version = "2.23", optional = true }
+data-encoding = "2.0.0-rc.1"
 itoa = "0.3"
 log = "0.3"
 ring = "0.7"
-rustc-serialize = "0.3"
 serde = "0.9"
 serde_derive = "0.9"
 serde_json = "0.9"
-url = "1.4"
 untrusted = "0.3"
+url = "1.4"
+
+[dependencies.chrono]
```

Comment:
Also can you leave this in the JSON-esque style and not TOML style? I find it easier to skim my dependencies that way.